example/lib/peripheral_list/peripheral_list_bloc.dart

```diff
@@ -52,11 +55,18 @@ class PeripheralListBloc
       updatedPeripherals = List.from(state.peripherals);
       updatedPeripherals.add(event.peripheral);
     }
-    yield PeripheralListState(
+    yield PeripheralList(
         peripherals: updatedPeripherals,
         scanningEnabled: state.scanningEnabled);
   }
 
+  Stream<PeripheralListState> _mapPickPeripheralToState(
+      PickPeripheral event) async* {
+    _blePeripheralRepository.blePeripheral = event.peripheral;
+    yield NavigateToPeripheralDetails(
```

Comment:
`NavigateToPeripheralDetails` requires a list of peripherals etc. it looks a little bit weird

Reply:
I agree, the implementation was changed since and whole new NavigatorBloc was introduced to handle navigation changes.